src/Tribe/Load_Assets.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Class used to hook into the assets being loaded
+ *
+ * @since TBD
+ */
+class Tribe__Load_Assets {
+
+	/**
+	 * Filter to listen when a tag is attached to the HTML.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		add_filter( 'script_loader_tag', array( $this, 'prevent_underscore_conflict' ), 10, 2 );
+	}
+
+	/**
+	 * Before underscore is loaded to the FE we add two scripts on before and one after to prevent underscore from
+	 * taking place on the global namespace if lodash is present.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $tag The <script> tag for the enqueued script.
+	 * @param string $handle The script's registered handle.
+	 * @return string The <script> tag.
+	 */
+	public function prevent_underscore_conflict( $tag, $handle ) {
+		if ( is_admin() || ! defined( 'TRIBE_EVENTS_FILE' ) ) {
+			return $tag;
+		}
+
+		if ( 'underscore' === $handle ) {
+			$path = plugin_dir_path( TRIBE_EVENTS_FILE ) . 'common/src/resources/js';
+			$dir = plugin_dir_url( TRIBE_EVENTS_FILE ) . 'common/src/resources/js';
```

Comment:
Use the base URL for common, because it might be loaded from Event Tickets and this would fatal.

Reply:
Perfect, but there's a check above to prevent errors if constant is not defined it will not be fired. Thanks.